Framework/DataHandling/test/LoadInstrumentTest.h

```diff
@@ -293,10 +294,59 @@ class LoadInstrumentTest : public CxxTest::TestSuite {
     AnalysisDataService::Instance().remove(wsName);
   }
 
+  /// Test the Nexus geometry loader from file
+  void testExecNexusLOKI() {
+    LoadInstrument loaderLOKI;
+    TS_ASSERT_THROWS_NOTHING(loaderLOKI.initialize());
+
+    // create a workspace with some sample data
+    wsName = "LoadInstrumentTestLOKI";
+    Workspace_sptr ws =
+        WorkspaceFactory::Instance().create("Workspace2D", 1, 1, 1);
+    Workspace2D_sptr ws2D = boost::dynamic_pointer_cast<Workspace2D>(ws);
+
+    // put this workspace in the data service
+    TS_ASSERT_THROWS_NOTHING(AnalysisDataService::Instance().add(wsName, ws2D));
```

Comment:
Please do not use the `AnalysisDataService` in your tests. instead mark the `Algorithm::isChild(bool isChild)` as True. Then pass in your workspaces directly as `shared_ptr` objects. I can share some examples of this with you if you like.

Reply:
I got that from the NIMROD test further up. Should I also remove it from there?